### DISCUSSION TITLE
Adding more info data to z menu

### DISCUSSION THREAD

**Discussion by Archie-2021** [gokcehan/lf#1385] [Ideas]
I was thinking about adding more info about files in `z` key menu. Is there a way to add data to that. There is `mediainfo` program I use usually to get more info about media files for example. For example the length of the audio/video is a big interest for me. Is there a way to add such a special data?

**Comment by gokcehan** [comment]
@Archie-2021 I think I have seen people using such programs in their `previewer` scripts, but unfortunately it is not available as an `info` field.

**Comment by og900aero** [comment]
> I was thinking about adding more info about files in `z` key menu. Is there a way to add data to that. There is `mediainfo` program I use usually to get more info about media files for example. For example the length of the audio/video is a big interest for me. Is there a way to add such a special data?

I use this (put in the lfrc file):

`map z !clear; mediainfo "$f"`
